envelope_test.go

```diff
@@ -492,6 +492,40 @@ func TestParseHTMLOnlyInline(t *testing.T) {
 	}
 }
 
+func TestParseInlineMultipart(t *testing.T) {
+	msg := test.OpenTestData("mail", "inlinemultipart.raw")
+	e, err := enmime.ReadEnvelope(msg)
+	if err != nil {
+		t.Fatal("Failed to parse MIME:", err)
+	}
+
+	if len(e.Errors) != 0 {
+		t.Errorf("len(e.Errors) got: %v, want: 0", len(e.Errors))
+	}
+
+	want := "Simple text."
+	if !strings.Contains(e.Text, want) {
+		t.Errorf("Downconverted Text: %q should contain: %q", e.Text, want)
+	}
+
+	if len(e.Inlines) != 1 {
+		t.Error("Should have no inline, got:", len(e.Inlines))
```

Comment:
It looks like this test is expecting one inline, but error text indicates otherwise.